spider/quotes_spider.py

```diff
@@ -0,0 +1,20 @@
+import scrapy
+
+
+class QuotesSpider(scrapy.Spider):
```

Comment:
Since there will be multiple weather sites to scrape, and the main framework doesn't need to understand their details, it's better to define an interface which defines the common input/output data structure.